### PR TITLE
Organize PPF contract functions

### PR DESCRIPTION
**Pull request by facuspagnuolo**
A little bit of polishing following some rules we already agreed on other repos

### CODE REVIEW

File: lerna-debug.log [hunk 1]
```diff
@@ -0,0 +1,69 @@
+0 silly input [ 'test' ]
```
Comment (sohkai) [inline lerna-debug.log]: Can we remove this file 😄?

File: packages/ppf-contracts/contracts/PPF.sol [hunk 2]
```diff
@@ -47,27 +47,67 @@ contract PPF is IFeed, TimeHelpers {
...
+        require(msg.sender == operatorOwner, ERROR_CAN_NOT_SET_OPERATOR_OWNER);
+        _setOperatorOwner(_operatorOwner);
+    }
+
+    /**
+    * @param base Address for the base token in the feed
+    * @param quote Address for the quote token the base is denominated in
+    * @return XRT for base:quote and the timestamp when it was updated
+    */
+    function get(address base, address quote) public view returns (uint128, uint64) {
```
Comment (sohkai) [inline packages/ppf-contracts/contracts/PPF.sol]: Generally we put `view` functions after non-view functions.

We normally have a linting rule set up for this; I'm surprised we haven't set up linting for ppf at all 😅

File: lerna-debug.log (hunk 1 above)
Comment (facuspagnuolo) [reply lerna-debug.log]: 🤦‍♂ my bad

Comment (facuspagnuolo) [reply lerna-debug.log]: 🤦‍♂ my bad

File: packages/ppf-contracts/contracts/PPF.sol (hunk 2 above)
Comment (facuspagnuolo) [reply packages/ppf-contracts/contracts/PPF.sol]: sure, let's do that :)

Comment (facuspagnuolo) [reply packages/ppf-contracts/contracts/PPF.sol]: actually this one is an external function, we are overriding its visibility :/

**Pull request merged**